CHANGELOG.md

```diff
@@ -4,24 +4,33 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
-### Handle Granules with Identical producerGranuleId in Different Collections
+## Handle Granules with Identical producerGranuleId in Different Collections
 
 - **CUMULUS-4059**
   - Added new non-null column `producer_granule_id` to Postgres `granules` table.
   - Added `producerGranuleId` property to `granule` record schema.
   - Updated `@cumulus` api/db/message packages to handle `producer_granule_id` and `producerGranuleId`.
   - Updated `@cumulus/api/lib/writeGranulesFromMessage` to set producerGranuleId = granuleId if not set.
   - Updated `queue-granules` task to set producerGranuleId = granuleId if not set.
+- **CUMULUS-4073**
+  - Adds AddUniqueGranuleId task to `ingest` terraform module for deployment with Core.
+  This task will update a payload of existing granules to have 'uniquified' IDs and preserve the original
+  identifier in the `producerGranuleId` field
+- **CUMULUS-4061**
+  - Added GenerateUniqueGranuleId to @cumulus/ingest for use in generating a hashed/'uniquified' granuleID
+- **CUMULUS-4073**
+  - Adds AddUniqueGranuleId task to `ingest` terraform module for deployment with Core.
+  This task will update a payload of existing granules to have 'uniquified' IDs and preserve the original
+  identifier in the `producerGranuleId` field
+- **CUMULUS-4061**
+  - Added GenerateUniqueGranuleId to @cumulus/ingest for use in generating a hashed/'uniquified' granuleID
 
 ## [Unreleased]
```

Comment:
Does this need to be moved up?

Reply:
No, I don't think so.  @jennyhliu  added it in her PR deliberately ahead of Unreleased to make merging easier.

Reply:
Yeah. We can always update when we merge the feature branch.
Nit: Add the entries order by ticket numbers to reduce merge conflicts.